modules/installation-osp-about-kuryr.adoc

```diff
@@ -5,18 +5,18 @@
 [id="installation-osp-about-kuryr_{context}"]
 = About Kuryr
 
-Kuryr is a container network interface (CNI) plug-in that uses Neutron and Octavia to provide networking to Pods and Services. It is primarily designed for {product-title} clusters that run on OpenStack instances.
+Kuryr is a container network interface (CNI) plug-in that uses Neutron and Octavia to provide networking to Pods and Services. It is primarily designed for {product-title} clusters that run on Red Hat OpenStack Platform (RHOSP) instances.
 
-Kuryr improves network performance by connecting Pods to the OpenStack SDN. It also provides connections between Pods and OpenStack instances.
+Kuryr improves network performance by connecting Pods to the OpenStack SDN. It also provides connections between Pods and RHOSP instances.
 
 [IMPORTANT]
 ====
-Kuryr is recommended for deployments on encapsulated OpenStack tenant networks to avoid double encapsulation. For example, on a deployment of an encapsulated OpenShift SDN over an OpenStack network.
+Kuryr is recommended for deployments on encapsulated RHOSP tenant networks to avoid double encapsulation. For example, on a deployment of an encapsulated OpenShift SDN over a RHOSP network.
```

Comment:
s/RHOSP tenant/RHOSP-tenant (assuming this is similar to a "single-tenant network")